neptune/new/attributes/namespace.py

```diff
@@ -61,10 +61,7 @@ def assign(self, value: Union[NamespaceVal, dict, Mapping], wait: bool = False):
             value = NamespaceVal(value)
 
         for k, v in value.value.items():
-            if k in self:
-                self[k].assign(v, wait)
-            else:
-                self._run.define(f"{self._str_path}/{k}", v)
+            self._run[f"{self._str_path}/{k}"].assign(v, wait)
```

Comment:
In general it was dup with a logic of Handler: https://github.com/neptune-ai/neptune-client/blob/1a18ba8eb2eaa5d5d707ddbe1df27edc1302fd66/neptune/new/handler.py#L101